recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
     patches:
         - clang.patch  # [osx]
         - prepend-dll.patch  # [win]
+        - disable_jpeg12.patch  # [win]
```

Comment:
> Yes. And then you can commit with [skip ci] since there are no changes to the build.

I didn't get this. Where should I have the [skip ci]? In the commit message?